client/sw.js

```diff
@@ -1,17 +1,41 @@
 /* eslint-disable */
 
 self.addEventListener('push', function (event) {
-  var data = {};
+
+  var title = "";
```

Comment:
var title; is enough
or we can declare these variables when it is actually used.